src/cgame/default/cg_types.h

```diff
@@ -86,4 +86,3 @@ typedef struct cg_particles_s {
 #endif /* __CG_LOCAL_H__ */
 
 #endif /* __CG_TYPES_H__ */
-
```

Comment:
Yep, it does. I was going to amend this commit without this change, but git was complaining.